site/en/tutorials/load_data/images.ipynb

```diff
@@ -0,0 +1,1356 @@
+    },
+    {
+      "metadata": {
+        "id": "HEsk5nN0vyeX",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "for label in label_ds.take(10):\n",
+        "  print(label_names[label.numpy()])"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "jHjgrEeTxyYz",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "Since the datasets are in the same order we can just zip them together to get a dataset of `(image, label)` pairs."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "AOEWNMdQwsbN",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "image_label_ds = tf.data.Dataset.zip((image_ds, label_ds))"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "yA2F09SJLMuM",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "The new dataset's `shapes` and `types`, are tuples of shapes and types as well, describing each field:"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "DuVYNinrLL-N",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "print('image shape: ', image_label_ds.output_shapes[0])\n",
+        "print('label shape: ', image_label_ds.output_shapes[1])\n",
+        "print('types: ', image_label_ds.output_types)\n",
+        "print()\n",
+        "print(image_label_ds)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "metadata": {
+        "id": "vYGCgJuR_9Qp",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "### Basic methods for training"
+      ]
+    },
+    {
+      "metadata": {
+        "id": "wwZavzgsIytz",
+        "colab_type": "text"
+      },
+      "cell_type": "markdown",
+      "source": [
+        "To train a model with this dataset you will want the data:\n",
+        "\n",
+        "* To be well shuffeled.\n",
+        "* To be batched.\n",
+        "* To repeat forever.\n",
+        "* Batches to available immediately when needed.\n",
+        "\n",
+        "These features can be easily added using the `tf.data` api."
+      ]
+    },
+    {
+      "metadata": {
+        "id": "uZmZJx8ePw_5",
+        "colab_type": "code",
+        "colab": {}
+      },
+      "cell_type": "code",
+      "source": [
+        "BATCH_SIZE = 32\n",
+        "\n",
+        "# Setting a shuffle buffer size larger than the dataset ensures that the data is completely shuffled.\n",
+        "ds = image_label_ds.repeat()\n",
+        "ds = ds.shuffle(buffer_size=4000) \n",
+        "ds = ds.batch(BATCH_SIZE)\n",
+        "# `prefecth` lets the dataset fetches batches, asynchronously while the model is training.\n",
```

Comment:
s/fetches batches, asynchronously/fetch batches asynchronously/

Reply:
s/prefecth/prefetch/

Reply:
Actually, for a novice user, I wonder if "in the background" would be more intuitive than "asynchronously"?